components/education/LearningPathCard.tsx

```
@@ -85,34 +104,32 @@ export function LearningPathCard({
           </div>
         </div>
       </CardHeader>
-      <CardContent className="flex-grow">
-        <div className="space-y-3">
+      <CardContent className="flex-grow px-3 sm:px-6">
+        <div className="space-y-2 sm:space-y-3">
           {modules.map((module, index) => (
             <div
               key={index}
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)